ArcadiaTeamsBot.CQRS/Handlers/GetServiceDeskRequestTypesHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ArcadiaTeamsBot.CQRS.Queries;
+using MediatR;
+using ServiceDesk.Abstractions.DTOs;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace ArcadiaTeamsBot.CQRS.Handlers
+{
+    class GetServiceDeskRequestTypesHandler : IRequestHandler<GetServiceDeskRequestTypesQuery, IEnumerable<ServiceDeskRequestTypeDTO>>
+    {
+        private const string requestTypesUrl = "https://arcadia-servicedesk-dev31.azurewebsites.net/api/intra/requestTypes";
+
+        public GetServiceDeskRequestTypesHandler(IHttpClientFactory clientFactory)
+        {
+            ClientFactory = clientFactory;
+        }
+
+        public IHttpClientFactory ClientFactory { get; }
```

Comment:
Again - why did you create this as public property?

Reply:
Made it private

Reply:
And why do you need private property?

Reply:
Made it private readonly field